### DISCUSSION TITLE
help recovering deleted files from dupeguru

### DISCUSSION THREAD

**Discussion by HelpPlease9** [hsoft/dupeguru#986] [Q&A]
I know nothing about coding.  I have a Win 10 PC.  I used dupeguru to scan my whole HD.  I created a folder on my EX HD to save the files.  Now, once the scan is done, I cannot open File Explorer.  That being the case, I am looking for a way via dupeguru to restore all of my files.  How can I do this?  Thanks.  PS:  I thought of going to my 'restore point' with the Win 10 OS - (I created a restore point just before I did the scan) but I do not know if that would fix the problem or make it worse.

**Comment by hrxn** [comment]
You have accidentally deduped your entire C: drive?

Forget it, just re-install Windows.

**Comment by arsenetar** [comment]
Base on what you said here, it seems like you used dupeGuru to delete/move the duplicates after the scan on you main hard drive?  Doing that on the whole drive would likely pick up Windows OS files, which you would not want to remove from the drive.  The only items I could suggest are trying to use the restore point or windows recovery tools from a USB/CD as it may not be possible to use the restore point if you cannot run things like File Explorer.

**Comment by HelpPlease9** [comment]
I am not sure what I have done.  I did Magoshare recovery and also a restore point.  I guess i'm lucky.  Things seem to be working again.  Thank God for Restore Points.  I didn't understand Megoshare either.  Bottom line for me, don't mess with tools that I do not understand.  Thanks.  Dean
    On Tuesday, March 29, 2022, 05:55:01 PM CDT, Andrew Senetar ***@***.***> wrote:  
 
 


Base on what you said here, it seems like you used dupeGuru to delete/move the duplicates after the scan on you main hard drive? Doing that on the whole drive would likely pick up Windows OS files, which you would not want to remove from the drive. The only items I could suggest are trying to use the restore point or windows recovery tools from a USB/CD as it may not be possible to use the restore point if you cannot run things like File Explorer.
